src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -455,7 +471,7 @@ function wgslValuesArray(
   expressionBuilder: ExpressionBuilder
 ): string {
   // AbstractFloat values cannot be stored in an array
```

Comment:
They can.

```
const an_array_of_abstract_float = array(1.0, 2.0, 3.0);
```

But like all abstract numeric types, they'll be concreted before being assigned to a `var`, `let` or parameter.